api/utils/helpers.js

```diff
@@ -19,9 +19,19 @@ const mapJSFiles = (dirname, mapFn, exclude = []) => {
 
 const classMixin = (A, B) => class A extends B {}
 
+const pack = (msg) => {
+  return JSON.stringify(msg)
+}
+
+const unpack = (msg) => {
+  return JSON.parse(msg)
+}
+
```

Comment:
I think these helpers only obstruct the meaning of the code but you may disagree

Reply:
Maybe for the moment. I think it is ok to change the semantics of the code having in mind that maybe in the future you will want to add extra logic inside this functions. Right ?

Reply:
I agree with Kostis that it's simpler to just write `JSON.stringify` and `JSON.parse` at this point. But even if you want to abstract them, do not call them `pack`/`unpack`. There is more standard terminology here such as "stringify" / "unstringify" or "marshal" / "unmarshal" or "serialize" / "unserialize" (the latter one being the most common in JS lingo). Instead, "pack" / "unpack" typically refers to particular encodings (e.g. Haskell's `pack` converts between `Word8` datatypes, which is simply a coercion, not serialization).